fence/auth.py

```diff
@@ -1,23 +1,27 @@
-import flask
-from flask_sqlalchemy_session import current_session
+import urllib.error
+import urllib.parse
+import urllib.request
+import flask 
+import binascii
+from configparser import RawConfigParser
 from functools import wraps
-import urllib.request, urllib.parse, urllib.error
+from os import environ
 
-from authutils.errors import JWTError, JWTExpiredError
-from authutils.token.validate import (
-    current_token,
-    require_auth_header,
-    set_current_token,
-    validate_request,
-)
-from cdislogging import get_logger
+from Crypto.PublicKey import RSA
+from Crypto.Signature import pkcs1_15
+from Crypto.Hash import SHA256
 
-from fence.errors import Unauthorized, InternalError
+from authutils.errors import JWTError, JWTExpiredError
+from authutils.token.validate import (current_token, require_auth_header,
+                                      set_current_token, validate_request)
+from flask_sqlalchemy_session import current_session
+from fence.config import config
+from fence.errors import InternalError, Unauthorized
 from fence.jwt.validate import validate_jwt
-from fence.models import User, IdentityProvider, query_for_user
```

Comment:
We need to keep Fence in sync with the master, it is easier adding things than editing/removing part. Please reduce changes to imports that are not necessary.

Reply:
Nothing was removed.  The order of the imported items changed
from:
`from fence.models import IdentityProvider, User, query_for_user`
to:
`from fence.models import User, IdentityProvider, query_for_user`

I reverted to the original order